<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingSchedule</class>
 <widget class="QWidget" name="TrainingSchedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>551</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Epochs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="Epochs"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Learning Rate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="LearningRate">
        <property name="decimals">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Batch Size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="BatchSize">
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Learning Rate Reduction</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="LearningRateReduction">
        <property name="decimals">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Learning Rate Reduction Frequency</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="LearningRateFrequency"/>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="SaveBest">
      <property name="text">
       <string>Save Best Perceptron</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="SaveLog">
      <property name="text">
       <string>Save Log</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="AbobaView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorWindow">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>550</y>
     <width>561</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
